### PR TITLE
Support alert manager

### PR DESCRIPTION
**Pull request by BoTranVan**

### CODE REVIEW

File: common.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>
+
+package gobizfly
+
+// SliceContains - Check data in slice
+func SliceContains(slice interface{}, val interface{}) (int, bool) {
+	switch v := slice.(type) {
+	case string:
```
Comment (cuonglm) [inline common.go:23]: Function name `SliceContains` but check for type `string` make no sense.

Please write separated functions for slice and string.

File: autoscaling.go [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+		LaunchConfigurations []*LaunchConfiguration `json:"profiles"`
+	}
+
+	if err := json.NewDecoder(resp.Body).Decode(&data); err != nil {
+		return nil, err
+	}
+
+	for _, LaunchConfiguration := range data.LaunchConfigurations {
+		if LaunchConfiguration.OperatingSystem.Error != "" {
+			LaunchConfiguration.Status = "Error"
```
Comment (greatbn) [inline autoscaling.go:167]: Use constant for these strings.

File: autoscaling.go [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+	defer resp.Body.Close()
+
+	data := &LaunchConfiguration{}
+
+	if err := json.NewDecoder(resp.Body).Decode(&data); err != nil {
+		return nil, err
+	}
+
+	if data.OperatingSystem.Error != "" {
+		data.Status = "Error"
```
Comment (greatbn) [inline autoscaling.go:218]: ditto

File: server_test.go [hunk 4]
```diff
@@ -207,6 +207,7 @@ func TestServerCreate(t *testing.T) {
 		Type:             "premium",
 		AvailabilityZone: "HN1",
 		OS:               &ServerOS{"cbf5f34b-751b-42a5-830f-6b2324f61d5a", "image"},
+		DataDisks:        []*ServerDisk{{40, "HDD"}},
```
Comment (greatbn) [inline server_test.go:210]: Why do you make a change here?

**Pull request merged**